### DISCUSSION TITLE
PrusaSlicer 2.6

### DISCUSSION THREAD

**Discussion by megatech1966** [supermerill/SuperSlicer#3492] [General]
When will Super Slicer be built using PS 2.6
It supports tree supports👍

**Comment by Garryb123** [comment]
You mean 2.5? Where is 2.6?

**Reply by cron410** [reply]
There's a 2.6 alpha with "Organic supports" which is likely what OP is referring to.

**Reply by Garryb123** [reply]
Daft me; where is that alpha?

It's not here:
https://github.com/prusa3d/PrusaSlicer/releases

**Reply by Garryb123** [reply]
Ah, it's not available yet:
https://www.reddit.com/r/prusa3d/comments/zfhbfp/prusa_26_download/

**Reply by agarza647** [reply]
A binary is not available, but the source is at [https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.6.0-alpha0](https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.6.0-alpha0) along with [build instructions here](https://github.com/prusa3d/PrusaSlicer#development).

**Reply by etet100** [reply]
Organic supports are not included in alpha0. I have built it. Prusa apparently decided not to release the most recent changes.

**Reply by CODeRUS** [reply]
> A binary is not available, but the source is at https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.6.0-alpha0 along with [build instructions here](https://github.com/prusa3d/PrusaSlicer#development).

you're wrong

**Comment by tome9111991** [comment]
Livestream PS2.6
https://youtu.be/ThiuzonYH10?t=1590

**Comment by phajduk** [comment]
Yeah, it would be great to have tree supports in SuperSlicer but... check this convo https://github.com/supermerill/SuperSlicer/discussions/2891 

I am afraid we will be forced to back to PrusaSlicer as SS seems to be an abandoned project :(

**Reply by Garryb123** [reply]
It's not dead. Please join Discord and the SuperSlicer channel.

**Reply by ChaosBlades** [reply]
What is the invite link? The only one I found was invalid.

**Reply by CODeRUS** [reply]
> What is the invite link? The only one I found was invalid.

https://github.com/supermerill/SuperSlicer/issues/611

**Comment by Beta1492** [comment]
PrusaSlicer 2.6.0-alpha2 officially dropped for the public! Any idea how long it will take to be implemented?

https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.6.0-alpha2
